js/config/firstWordExceptions.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Consider moving this file into a separate subdirectory, dedicated purely to the English language.